modules/cnf-du-partitioning-management-workloads.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-partitioning-management-workloads_{context}"]
+
+= Partitioning management workloads
+
+You can isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved
+set of CPUs. This is useful for resource-constrained environments, such as single-node production deployments,
+where you want to reserve most of the CPU resources for user workloads and configure OpenShift to run on a fixed
+number of CPUs within the host.
+
+Server resources deployed at the edge, such as cores and memory, are expensive and limited. Customers require nearly all cores for their application workloads. The resources consumed by infrastructure is a key portion of the customer's business case for the selection of a vRAN infrastructure.
+
+A hypothetical DU example is an unusually resource-intensive workload, typically requiring 20 dedicated cores, 24 GiB of RAM consumed as huge pages, multiple SR-IOV NICs carrying several Gbps of traffic each, and specialized accelerator devices. The node hosting this workload must run a realtime kernel, and be carefully tuned to ensure low-latency requirements can be met, and be configured to support features such as Precision Timing Protocol (PTP).
+
+This feature implements the necessary OCP features and cluster configuration to isolate all housekeeping and infrastructure pods to a partitioned set of CPU resources on a Single Node OpenShift (SNO) cluster with a DU profile applied. This falls broadly into two categories:
+
+* Isolates cluster management functions to the defined number of CPUs. All housekeeping tasks and infrastructure pods operate solely on that cpuset.
+
+* Tunes the cluster configuration (with the applied DU profile) so the actual CPU usage fits within the assigned cpuset.
+
+[NOTE]
+====
+This feature is only available on SNO in this release.
+====
+
+The minimum number of reserved CPUs required for the management partition for an SNO is four CPUs/HTs. Inclusion of operators or workloads beyond the set specified in “Management pods” requires additional CPU/HTs.
```

Comment:
@browsell do we need to specify the type of system here, eg "is four CPUs/HTs of a modern high performance server"

Reply:
I think this is OK as is.